app/code/Magento/Tax/Model/System/Config/Source/Tax/Country.php

```diff
@@ -13,10 +13,9 @@ class Country extends \Magento\Directory\Model\Config\Source\Country
     protected $_options;
 
     /**
-     * @param bool $noEmpty
-     * @return array
+     * @inheritdoc
      */
-    public function toOptionArray($noEmpty = false)
+    public function toOptionArray($noEmpty = false, $foregroundCountries = '')
     {
         $options = parent::toOptionArray($noEmpty);
```

Comment:
Please add `$foregroundCountries` here in the call, it's supported by the parent.

Reply:
@buskamuza, done.